app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  has_many :comments, foreign_key: 'posts_id'
+  has_many :likes, foreign_key: 'posts_id'
+  belongs_to :user, foreign_key: 'users_id'
```

Comment:
- [ ] Your belongs_to should look something like below :arrow_down: because you have a user table which has a primary key author_id so it is a bit complicated here and can confuse the whole application because when you create a table suppose the user then rails expects it primary key user_id but here you have author_id so be careful with it :+1:

`belongs_to :author, class_name: 'User'`

Reply:
- [ ] **[REQUIRED]** Good job so far :clap:  But as the previous reviewer mentioned , The associations between your different models can be confusing because of user_id and author_id . Please make sure that it looks similar to the one below :arrow_down:  in order t make your application works as expected :+1: 

```ruby
belongs_to :author, class_name: 'User'
has_many :comments
has_many :likes
```